apps/xmtp.chat/src/helpers/names.ts

```diff
@@ -23,14 +23,18 @@ export const resolveName = async (name: string, force: boolean = false) => {
     return null;
   }
 
+  const normalizedName = name.toLowerCase();
+
   // check cached profiles
-  const cachedProfiles = profilesStore.getState().getProfilesByName(name);
+  const cachedProfiles = profilesStore
+    .getState()
+    .getProfilesByName(normalizedName);
   if (!force && cachedProfiles.length > 0) {
     return cachedProfiles;
   }
 
   const response = await fetch(
-    `${import.meta.env.VITE_API_SERVICE_URL}/api/v2/resolve/name/${window.encodeURIComponent(name)}`,
+    `${import.meta.env.VITE_API_SERVICE_URL}/api/v2/resolve/name/${window.encodeURIComponent(normalizedName)}`,
```

Comment:
the normalization can occur on the API service. doing it here doesn't really matter as we shouldn't trust clients.